node/go.mod

```diff
@@ -384,3 +384,5 @@ replace github.com/wormhole-foundation/wormchain => ../wormchain
 replace github.com/CosmWasm/wasmd v0.30.0 => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-2
 
 replace github.com/cosmos/cosmos-sdk => github.com/wormhole-foundation/cosmos-sdk v0.45.9-wormhole
+
+replace github.com/certusone/wormhole/node/pkg/txverifier => ./pkg/txverifier
```

Comment:
Why is this necessary when we don't do it for any of the other local packages?

Reply:
IMO we should do it for the other packages but I didn't want to make more changes than necessary.